deployments/gac/configurator.nix

```diff
@@ -0,0 +1,20 @@
+{ config , ...  }:
+
+with (import <nixpkgs> {}).lib;
```

Comment:
i believe `{ config, lib, ... }:` will also get you lib

Reply:
Are you sure? Does Nixops provide a lib argument? I know that `gac` doesn't (but maybe it should)

Reply:
Actually it doesn't look like this `with` is used at all